.gitignore

```diff
@@ -15,3 +15,4 @@ venv
 sqlite.db
 vagrant.yml
 .cache
+node_modules/
```

Comment:
We should also probably add these:
```
.deploy.lock
package-lock.json
package.json
```

Reply:
Yes, I didn't see that `npm` added files. This is done in latest force push.